app/code/Magento/Backend/view/adminhtml/templates/admin/access_denied.phtml

```diff
@@ -3,29 +3,31 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
-?>
-<?php
-/**
- * @see \Magento\Backend\Block\Denied
- */
+declare(strict_types=1);
+
+use Magento\Backend\Block\Denied;
+use Magento\Framework\Escaper;
+
+/** @var Escaper $escaper */
+/** @var Denied $block */
```

Comment:
It seems that the mention of the block is redundant here.

```suggestion
use Magento\Framework\Escaper;

/** @var Escaper $escaper */
```

Reply:
I'm personally a fan of keeping the `$block` reference - it helps people identify which block class the template is married to without having to dig through layout XML. I think keeping it adds value but am keen to hear your thoughts, @andrewbess 🙂